conf/nutch-default.xml

```diff
@@ -1566,6 +1566,16 @@ visit https://wiki.apache.org/nutch/SimilarityScoringFilter-->
   </description>
 </property>
 
+<!-- index-jexl-filter plugin properties -->
+
+<property>
+  <name>index.jexl.filter</name>
+  <value></value>
+  <description> A JEXL expression. If it evaluates to false,
```

Comment:
Probably we should add a comment here specifying which parameters are available for filtering. This would avoid checking the source code to see what you could use for filtering.

Reply:
Is this now addressed @jorgelbg ?